include/net/nrf_cloud_os.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_OS_H__
+#define NRF_CLOUD_OS_H__
+
+/** @defgroup nrf_cloud_os nRF Cloud OS specifics
+ * @{
+ */
+
+#include <stdlib.h>
+#include <zephyr/kernel.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Custom OS memory hooks for nRF Cloud lib */
+struct nrf_cloud_os_mem_hooks {
+	void *(*malloc_fn)(size_t size);
+	void *(*calloc_fn)(size_t count, size_t size);
+	void (*free_fn)(void *ptr);
+};
+
+/**
+ * @brief As a default nRF Cloud lib uses OS kernel heap memory
```

Comment:
Shouldn't this be the description of the nrf_cloud_os_mem_hookd_init function?

If so, you should maybe have something else before explaining about the default memory. Something like:
"Initialize the ....

Reply:
Changed